src/prototypes/draft-email-textarea/example/example.twig

```diff
@@ -0,0 +1,64 @@
+<div class="o-container o-container--pad o-container--prose">
+	<div class="o-container__content o-rhythm o-rhythm--default o-rhythm--generous-headings">
+
+		<textarea id="email-draft" class="c-input u-pad-1 js-elastic-textarea" spellcheck="false" rows="5">Hi there,
+
+My name is [Firstname Lastname], I’m a [role] at [organization].
+
+I’m reaching out to discuss [project, question, challenge]. [Additional details, if any.]
+
+Our budget for this effort is [budget, range, etc.] and we’re interested in wrapping up in [timeframe].
+
+I’m available to chat [potential meeting times], let me know if any of those work for you.
+
+Have a great day!
+
+[Firstname Lastname]</textarea>
+
+		<div class="o-button-group o-button-group--grow u-space-block-start-n2">
```

Comment:
Is there a reason we're using raw HTML here instead of Liquid templates?

Reply:
Oh, how do I do that from the WP Gutenberg editor?

Reply:
Oh, now I follow! You asked why I didn't use Liquid templates _in the prototype_. My bad, I misunderstood. LOL

My brain was stuck in WP admin land so I did a straight copy + paste of the code I had over there. I never considered using Liquid templates in the prototype because it just didn't occur to me, but I should!